src/main/java/com/api/parser/ParserApplication.java

```diff
@@ -3,11 +3,21 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import java.util.Arrays;
+
 @SpringBootApplication
 public class ParserApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(ParserApplication.class, args);
+
+		//Test
+		JSONParser JSONparser = new JSONParser();
+//		Parser XMLparser = new XMLParser();
+//		String[] json = JSONparser.getTag86Parts();
+//		String xml = XMLparser.parseMT940();
+		System.out.println(JSONparser.parseMT940());
+//		System.out.println(xml);
```

Comment:
de test kan verplaatst worden naar een apart test bestand, of anders even weg halen